core/capabilities/ccip/ccipaptos/utils.go

```diff
@@ -0,0 +1,23 @@
+package ccipaptos
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	"github.com/smartcontractkit/chainlink-integrations/evm/utils"
+)
+
+func abiEncodeUint32(data uint32) ([]byte, error) {
+	return utils.ABIEncode(`[{ "type": "uint32" }]`, data)
+}
+
+func abiDecodeUint32(data []byte) (uint32, error) {
```

Comment:
Duplicated from elsewhere, consolidate if needed, but again kinda confused about the ABI decoding going on.